### ISSUE TITLE
Insufficient color contrast on link "underline" (border)

### ISSUE THREAD

**Issue by erinchambers** [CDRH/lewisandclark#256]
A thick `border-bottom` like we have now is a totally valid strategy for the accessibility standard "Information (e.g. links) must not be distinguished by color alone." The current gray is too light, however, and has insufficient contrast against the background.

Note that this gray doesn't necessarily have to meet the 4.5:1 ratio for normal text contrast, so if meeting that ratio makes it awkwardly dark, that's okay—but it should definitely meet the 3:1 for graphical objects and user interface components (see the [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/) for testing).
